addons/point_of_sale/static/src/app/models/product_template.js

```diff
@@ -118,7 +118,7 @@ export class ProductTemplate extends Base {
         return (
             this.isConfigurable() &&
             this.attribute_line_ids.length > 0 &&
-            !this.attribute_line_ids.every((l) => l.attribute_id.create_variant === "always")
+            this.attribute_line_ids.some((l) => l.attribute_id.create_variant === "no_variant")
```

Comment:
Also check if there is several possible values

```suggestion
            this.attribute_line_ids.some((l) => l.attribute_id.create_variant === "no_variant" && l.product_template_value_ids.length > 1)
```

Reply:
Yes but the method is "needtoconfigure" make more sense to check here